srv/components/csm/config/config.sls

```diff
@@ -15,6 +15,19 @@
 # please email opensource@seagate.com or cortx-questions@seagate.com.
 #
 
+
+Register features to be unsuported:
+  module.run:
+    - cortx_unsupported_features.register:
+      - component: provisioner
+      - unsupported_feature_list:
+        - fw_update
+      - setup_types:
+        - VM
```

Comment:
Since the setup type RBOD is missing here, does that mean the fw update will be supported only for RBOD?

Reply:
Yes for now we should maintain support for RBOD. But once it is officially dropped from our supported features list, we can disable it for RBOD as well by listing here.

Reply:
I have made the addition to `test/r2.md` for manual testing.